app.js

```diff
@@ -53,8 +53,9 @@ function addClass(insertedDiv, letter, i) {
 function gameSetup() {
   addLetterListeners();
   insertNextImg();
-  let rndPickedProverb = proverbs[getRandomInt(proverbs.length)];
-  sentence.innerHTML = encryptSentence(rndPickedProverb);
+  state.proverb = proverbs[getRandomInt(proverbs.length)].toLowerCase();
```

Comment:
you should add initial value of proverb to state object at the beginning of this file